news/2021-05-07-monthly-beatmapping-contest-may-2021.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Monthly Beatmapping Contests: May 2021"
+date: 2021-05-07 08:00:00 +0000
+---
+
+Try your hand at this month's slow-slider-themed contest with a chance to become osu!'s next *Elite Mapper*!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+May's [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) (*MBC*) includes opportunities for [**osu!**](LINK), [**osu!taiko**](LINK), and [**osu!mania**](LINK) mappers. Maybe next month we'll cover every mode at the same time, who knows?
+
+## Schedule
+
+- **Beatmapping (2.5 weeks):** Contests are announced at the start of the month and beatmappers create their entries.
+- [**Screening (1.5 weeks):**](/wiki/Contests/Monthly_Beatmapping_Contest#screening) Competitive entries are selected by experienced beatmappers for each mode.
+- [**Judging (1.5 weeks):**](/wiki/Contests/Monthly_Beatmapping_Contest#judging) Competitive entries are judged by different experienced beatmappers for each mode.
+- [**Rewards:**](/wiki/Contests/Monthly_Beatmapping_Contest#rewards) Winners are announced and prizes are redeemed, including fancy profile badges and the legendary *Elite Mapper* title!
+
+Expert mathematicians might notice that this whole schedule doesn't fit within a month. We announce results for each contest halfway through the following month, meaning May's contest results will be published mid-June.
+
+## Beatmapping
+
+### <a id="limitation"></a>Limitation
+
+We have two goals here at MBC headquarters: to create mapping contests every month and to ensure those contests aren't repetitive. It's a tough act to balance, which is why we have **limitations** — mapping restrictions that submissions need to follow, like "straight sliders only" or "circles cannot be used" or "you must map with your eyes closed". We unfortunately haven't tried that last one yet.
+
+Below are the limitations for this month!
+
+**[osu! MBC](LINK) limitations:**
+
+- Total slider velocity (SV) cannot be higher than **1.4**
+
+What's "total slider velocity"? It's not an official term, but we're talking about a map's **base SV** (set in the Timing tab of the editor) combined with any **SV multipliers** (green lines set in the F6 timing panel).
+
+So for example:
+
+- If base SV is **1.4**, SV multipliers can't be higher than **1x**. (1.4 x 1 = 1.4 "total SV")
+- If base SV is **2.8**, your multipliers can't be higher than **0.5x** (2.8 x 0.5 = 1.4 "total SV").
+
+Pretty straightforward, right? For those who'd rather not think, the default base SV is 1.4, so don't touch that and only use SV multipliers below 1x. You'll be fine.
```

Comment:
```suggestion
Pretty straightforward, right? For those who'd rather not think, the default base SV is 1.4, so don't touch that and only use SV multipliers below 1x and you'll be fine.
```

Reply:
Previous version reads better. Your change brings another "and" into a sentence which already has "and" in it.

Reply:
I struggle to find any source that says that you cannot use "and" in a sentence multiple times. I agree that the current usage can sound a bit weird, but having a full-stop right before "You'll be fine" makes it sound way off because it suggests that it is completely separate from the previous sentence.

We *could* add commas in to make the sentence read as `For those who'd rather not think, the default base SV is 1.4, so don't touch that, and only use SV multipliers below 1x, and you'll be fine.`, but then that just kicks the problem down from being and-a-licious to being comma-licious.

Either way, it's a suggestion for a reason; neither way is grammatically wrong.

Reply:
> having a full-stop right before "You'll be fine" makes it sound way off because it suggests that it is completely separate from the previous sentence.

i think people will assume the last sentence is related to the topic of the paragraph, so id rather stick with the current version since it's cleaner to read imo